app/views/exercise_solutions/_expectations.html.erb

```diff
@@ -0,0 +1,7 @@
+<% expectation = assignment.visible_expectation_results.find { |it| it[:status] != :passed } %>
```

Comment:
use `passed?` message: `!it[:status].passed?`. This abstract you from dealing with strings or symbols.

Reply:
PS: sorry for not detecting this before :pensive:

Reply:
`passed?` is true when status is passed or passed_with_warnings.

I need to be true only when is passed and false if passed_with_warnings

Reply:
Good point!

But I think this is only true for `Status` object -  which is wrong and inconsistent, I think - but not for strings or symbols - and I understand this is the case here, right?

```ruby
[1] pry(main)> :passed.passed?
=> true
[2] pry(main)> :passed_with_warnings.passed?
=> false
[3] pry(main)> 
```